### ISSUE TITLE
Do not hide the function argument hint when typing in the script editor.

### ISSUE THREAD

**Issue by SilencedPerson** [godotengine/godot-proposals#3897]
### Describe the project you are working on

3D game.

### Describe the problem or limitation you are having in your project

I started writing out this function:
`move_and_slide_with_snap(motion,Vector3(0,grounded,0),Vector3.UP,true,4, *)`
* there is some float which that get's hidden as soon as i hit any key on the keyboard thus making the hint, completely useless.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Users will be able to type in arguments that are already available in the hint without having to resort to googling them.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Make it persistent while the cursor/caret is on the line with built in function.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No.

### Is there a reason why this should be core and not an add-on in the asset library?

Text editor is part of a core.

**Comment by Calinou** [comment]
The callhint is displayed as long as there's still at least one valid suggestion. If there is no valid suggestion, it should _not_ linger on. This is the same behavior as in most other text editors and IDEs.

**Comment by SilencedPerson** [comment]
![wsPtt2oyqA](https://user-images.githubusercontent.com/82024211/151790820-ea2eed03-f016-4a26-ac08-700a1e3b226d.gif)
It happens so often to me, it's really annoying.
for example, vs studio code does it like this, where the callhint is displayed above the function while the autocomplete is below.
![image](https://user-images.githubusercontent.com/82024211/151791742-8ef8adfd-a68d-4479-a2bc-0a8e3f7543a2.png)

**Comment by Zireael07** [comment]
Yep, I've had the call hint flickering on and off when writing some code recently, was a big annoyance as it was some function I wasn't familiar with and I had to look it up manually in the help to know what it wanted.

Even worse, it does the same for my custom functions which only appear in help in Godot 4, so no workaround for Godot 3.x
